packages/components/src/tag/src/Tag.css

```diff
@@ -139,10 +139,10 @@ a.o-ui-tag-solid.o-ui-tag-disabled {
 
 button.o-ui-tag-outline::after,
 a.o-ui-tag-outline::after {
-    top: -1px;
-    left: -1px;
-    right: -1px;
-    bottom: -1px;
+    top: -3px;
```

Comment:
Is this possible to have a calc here so we better understand where the 3px comes from?

Reply:
You could use inset instead.